RHDHPAI-808: Developer Lightspeed content RHDH (#1300)
redhat-developer/red-hat-developers-documentation-rhdh: modules/developer-lightspeed/con-your-responsiblity.adoc

:_mod-docs-content-type: CONCEPT
[id="con-your-responsibility_{context}"]
= Your responsibility

> All of the information your users share in their questions and responses with {ls-short} are shared with the LLM inference service you configured. It is your responsibility to ensure that you follow your company's relevant policies regarding the sharing of data with your chosen inference service. 

Reviewer: ```suggestion
All information that your users share in their questions and responses with {ls-short} is shared with the LLM inference service you configured. You are responsible for ensuring compliance with your company's policies regarding the sharing of data with your chosen inference service. 
```
